core/test/qa_Block.cpp

```diff
@@ -557,8 +556,8 @@ stride_test(const StrideTestData &data, std::shared_ptr<gr::thread_pool::BasicTh
     if (data.in_port_min >= 0) int_dec_block.in.min_samples = static_cast<size_t>(data.in_port_min);
 
     expect(eq(gr::ConnectionResult::SUCCESS, flow.connect<"out">(source).to<"in">(int_dec_block)));
-    auto sched = scheduler(std::move(flow), thread_pool);
-    sched.runAndWait();
+    auto sched = scheduler(std::move(flow), std::move(thread_pool));
+    expect(sched.runAndWait().has_value());
```

Comment:
Not for this PR, but I think we need to add this `expect(sched.runAndWait().has_value());` for all test, potentially it saves debugging time as it was the case here.